cygrid/cygrid.pyx

```diff
@@ -976,7 +993,7 @@ cdef class SlGrid(Cygrid):
             self,
             np.ndarray[double, ndim=1] sl_lons,
             np.ndarray[double, ndim=1] sl_lats,
-            long naxes3,
+            long naxes3=1,
```

Comment:
I'm a bit confused by the naxes3 keyword. Shouldn't this be naxis3 or naxes? And why exactly is the default value 1? Or is this meant to be boolean (ie data is cube if naxes3=True, map if naxes3=False)?